content/account_management/faq.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Account management FAQ
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+---
+
+### How do I setup my team in Datadog?
+
+The admin of the account should enter the email addresses of team members
+[here][other-1]. Some team best practices are as follows:
+
+* When the team member receives the confirmation email, they will be provided with a link to log in directly. The user should not click ‘sign up’ during this process.
+* If multiple users from the same organization sign up separately, this will register as different organizations in Datadog. Please reach out to support to have these merged, but please note that all information contained in the account getting merged will not be transferred over.
+* The only access controls we have right now are around admin activities (adding/removing users, billing, etc.). As far as data goes (hosts, metrics, dashboards, etc.) all users have access to everything; more robust access controls are in our pipeline, but not something we’ve focused a lot of attention on yet.
```

Comment:
Users can be placed in one of three permissions categories which include Read Only, Standard, and Administrator, Standard User and Read Only.   All three categories are able to view metrics, monitors, dashboards and other account data.  

Read Only users are able to all data, excluding billing information, but will be unable to make changes to dashboards, monitors, or integrations.

Standard users have read and write access to all Datadog functions such as installation or configuration of integrations, dashboards, and monitor creation. Standard users do not have access to user management or billing workflows.

Administrators have access to all read only and standard user features, but may also manage/invite users in the Datadog organization, change subscription plans, and view billing reports.

Reply:
👍